lib/i18n/ocf-form.js

```diff
@@ -18,8 +18,7 @@ const i18nLabels = defineMessages({
     defaultMessage: 'What is the name of your initiative?',
   },
   slug: {
-    id: 'createCollective.form.slugLabel',
-    defaultMessage: 'Set your URL',
+    defaultMessage: 'Set your URL (This can be edited later)',
```

Comment:
Feedback was:
> Can we add a hover (?) saying "This can be edited later"

Should we add it as an over rather than to the label?

Reply:
Done, here's how it looks now. 

![Screenshot from 2021-11-19 09-08-59](https://user-images.githubusercontent.com/12435965/142663354-e61aa120-7549-4fb6-ada0-e97a4c30a4f2.png)